package.json

```diff
@@ -14,8 +14,8 @@
     "@fortawesome/free-brands-svg-icons": "^5.14.0",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "@gen3/guppy": "^0.15.0",
-    "@gen3/ui-component": "^0.11.0",
+    "@gen3/guppy": "uc-cdis/guppy#fix/dep",
+    "@gen3/ui-component": "uc-cdis/gen3-ui-component#fix/dep",
```

Comment:
I think we should release the new version instead of using the one in feature branch